codemeta.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+  "@type": "SoftwareSourceCode",
+  "license": "https://spdx.org/licenses/MIT",
+  "codeRepository": "https://github.com/openMetadataInitiative/openMINDS_MATLAB",
+  "contIntegration": "https://github.com/openMetadataInitiative/openMINDS_MATLAB/actions",
+  "issueTracker": "https://github.com/openMetadataInitiative/openMINDS_MATLAB/issues",
+  "name": "openMINDS MATLAB",
+  "description": "openMINDS MATLAB is a toolbox to support the creation and use of openMINDS metadata models and schemas in MATLAB, with import and export in JSON-LD format. The package contains all openMINDS schemas as MATLAB classes in addition to schema base classes and utility methods.",
+  "applicationCategory": "neuroscience",
+  "releaseNotes": "https://github.com/openMetadataInitiative/openMINDS_MATLAB/releases",
+  "funding": "https://cordis.europa.eu/project/id/945539",
```

Comment:
I assume that this is not enough (depending on the version). Right now you would receive funding through EBRAINS 2.0 I guess?

Reply:
should this not be an array? with funder inside? not sure how codemeta handles this...